packages/labs/testing/src/lib/utils.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+export const nextFrame = () =>
```

Comment:
Do we want to vend this? It's very basic and not specific to Lit.

Reply:
Are you asking if we should or should not vend it? Currently only the strip functions are exported directly from the index. The `nextFrame` would be accessible if the user digs into `lib/utils.js` though.

Reply:
Oh I see, that's fine.